.github/workflows/ci-actions-incremental.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches-ignore:
       - 'dependabot/**'
+      # TODO: revisit this in case incremental becomes the default
+      - 'master'
```

Comment:
I think as long as we have this incremental shadow CI, it's better to comment out the entire `push` section.

Reply:
I ended up *not* doing this because then pushes in forks would not trigger incremental build either, but IMO we want that during the testing phase.